fix: out of bounds error on mockCall (#1677)

